src/kakarot/accounts/account_contract.cairo

```diff
@@ -118,6 +122,40 @@ func __execute__{
     response_len: felt, response: felt*
 ) {
     alloc_locals;
+
+    // Upgrade flow
+    let (latest_account_class, latest_cairo1helpers_class) = AccountContract.get_latest_classes();
+    let (this_helpers_class) = Account_cairo1_helpers_class_hash.read();
+    if (latest_cairo1helpers_class != this_helpers_class) {
+        Account_cairo1_helpers_class_hash.write(latest_cairo1helpers_class);
```

Comment:
let's use `this_helpers_class`, `latest_helpers_class` and `Account_helpers_class` and remove everywhere the `cairo1` thing (wen cairo2)

Reply:
also, is `_class` or `_class_hash` the best prefix? no strong opinion but let's just choose one

Reply:
using _class